modules/va_notify/db/migrate/20241119031849_applies_modifications_to_va_notify_notifications.rb

```diff
@@ -0,0 +1,6 @@
+class AppliesModificationsToVANotifyNotifications < ActiveRecord::Migration[7.1]
+  def change
+    add_column :va_notify_notifications, :callback_klass, :text
```

Comment:
Please add `if_not_exists: true` to each add_column line.

Reply:
Added! Thanks!